app/serializers/project_serializer.rb

```diff
@@ -56,10 +56,6 @@ def student_id
     object.student.username
   end
 
-  def tutor_name
-    object.main_tutor.first_name unless object.main_tutor.nil?
-  end
-
```

Comment:
@macite Do we need this now?

Reply:
No - we have the tutorial enrolments... and we should be getting the name from the unit data.